src/app/Caches/Create/CreateCacheWizard.tsx

```diff
@@ -177,7 +255,47 @@ const CreateCacheWizard = (props) => {
         nextButtonText: 'Create Cache',
     };
 
-    const steps = [stepGettingStarted, stepEditCode];
+    const stepConfigure = {
+        name: 'Configuration',
+        steps: [
+            {
+                id: 3,
+                name: 'Substep A with validation',
+                component: (
+                    <p>Subset A</p>
+                ),
+                // enableNext: isFormValid,
+                // canJumpTo: stepIdReached >= 2
+            },
+            { id: 4, name: 'Substep B', component: <p>Substep B</p> }
+        ]
+    };
+
+    const steps = [stepGettingStarted,
+        ...(showEditCode ? [stepEditCode] : []),
+        ...(showConfiguration ? [stepConfigure] : [])];
+
+    const CustomFooter = (
+        <WizardFooter>
+            <WizardContextConsumer>
+                {({ activeStep, goToStepByName, goToStepById, onNext, onBack, onClose }) => {
+                    return (
+                        <>
+                            <Button variant="primary" type="submit" onClick={() => getNextStep(activeStep, onNext)}>
```

Comment:
the variants of the button are included in Patternfly constants I think. ButtonVariant. check other buttons